$NetBSD: patch-ac,v 1.1 2004/03/01 17:08:11 mpasternak Exp $

See comments in patches/patch-aa before removing this file. Thank you!

	-- dotz@irc.pl

--- app/tools/gimptexttool.c.orig	Mon Mar  1 17:50:58 2004
+++ app/tools/gimptexttool.c
@@ -187,7 +187,7 @@ gimp_text_tool_control (GimpTool       *
 			GimpToolAction  action,
 			GimpDisplay    *gdisp)
 {
-  GimpTextTool *text_tool = GIMP_TEXT_TOOL (tool);;
+  GimpTextTool *text_tool = GIMP_TEXT_TOOL (tool);
 
   switch (action)
     {
@@ -218,7 +218,7 @@ gimp_text_tool_button_press (GimpTool   
 			     GdkModifierType  state,
 			     GimpDisplay     *gdisp)
 {
-  GimpTextTool *text_tool = GIMP_TEXT_TOOL (tool);;
+  GimpTextTool *text_tool = GIMP_TEXT_TOOL (tool);
   GimpDrawable *drawable;
   GimpText     *text  = NULL;
   gint          off_x = 0;
